python/hsfs/feature_view.py

```diff
@@ -201,6 +201,10 @@ def init_serving(
                   For example: `{"pool_size": 10}`
         """
 
+        # initiate batch scoring server
+        # `training_dataset_version` should not be set if `None` otherwise backend will look up the td.
+        self.init_batch_scoring(training_dataset_version)
+
```

Comment:
I'm confused. Why is this necessary here? Here we are initializing the training dataset for the `get_feature_vector()` why are calling the `init_batch_scoring()`?

Reply:
I have the same thought, we should remove it. But for backward compatibility, I still put it here. I think before we wanted a single function to init both online and offline serving with the td version.